src/libraries/Common/src/Interop/OSX/Interop.libc.cs

```diff
@@ -27,5 +27,22 @@ internal struct AttrList
         internal static unsafe partial int setattrlist(string path, AttrList* attrList, void* attrBuf, nint attrBufSize, CULong options);
 
         internal const uint FSOPT_NOFOLLOW = 0x00000001;
+        [LibraryImport(Libraries.libc, EntryPoint = "fsetattrlist", StringMarshalling = StringMarshalling.Utf8, SetLastError = true)]
```

Comment:
This specifies Utf8 string marshaling. What string is being marshaled?

Reply:
/cc @tmds

Reply:
It looks like a holdover to me. 
We pass a handle instead of a path, after all.

@tmds Can you confirm this?

Reply:
You can just remove it, as there is no string

Reply:
This is a copy-paste from `setattrlist(string path` where it serves for the path marshalling. Here it has no use.

Reply:
okay 👍🏼